README.md

```diff
@@ -1,3 +1,103 @@
-# Docker Traceability Plugin
+# CloudBees Docker Traceability Plugin
 
-This plugin allows the tracking of the creation and use of Docker containers in Jenkins.
+This [Jenkins](http://jenkins-ci.org) plugin allows the tracking of the creation and use of Docker containers in Jenkins and their future use.
+
+#Plugin Summary
+
+* Container deployments summary page
+* Tracking of Docker image and container deployments produced in Jenkins
+* Tracking of Docker container events
+* Tracking of Docker container states being retrieved from *docker inspect* calls
+* Advanced [API](#API):
+ * Submission of events from Docker and Docker Swarm installations
+ * Data polling from Jenkins (including Docker API-compatible JSONs)
+ * Support of search queries
+ 
+ ![Main page](/doc/images/root-action.png)
+
+#Installation Guidelines
+
+##Plugin setup
+
+1. Install Docker Traceability plugin from Jenkins Update Center
+2. If you are going to use Docker Traceability for images built on Jenkins, consider the installation of other
+3. Configure the security. This step is **very important**, because the plugin can store raw JSON, which may contain sensitive info like passwords
+ * The plugin introduces new permissions, which allow to restrict the access
+ * **Read** - Allows to retrieve details like full container/info dumps. 
+      Web interfaces are being managed by common Jenkins Read permission
+ * **Submit** - Allows to submit deployment records from [API](#API)
```

Comment:
via **the** remote api.  (I assume its the remote API rather than the internal java API)